content/admin.py

```diff
@@ -62,6 +62,16 @@ class DonateSectionAdmin(admin.ModelAdmin):
     fields = ['category','order', 'color', 'title', 'img', 'text']
     list_display = ('order', 'category', 'title')
 
+class SubMenuInline(admin.TabularInline):
+    model = SubMenu
+    extra = 3
+
+
+class MenuAdmin(admin.ModelAdmin):
+    list_display = ('title', 'order')
+    fieldsets = [(None,{'fields': ['title']}),]
```

Comment:
yea sure, in all your other arrays and function calls, you're leaving a space after the comma, which is a good thing.

So it would look nicer if `[(None,{'fields': ['title']}),]` can turn into `[(None, {'fields': ['title']}),]`

Most style stuff that I talk about is in pep8, which is the default python style guide. https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements